app/models/manageiq/providers/openstack/inventory/persister/cloud_manager.rb

```diff
@@ -40,5 +40,14 @@ def initialize_inventory_collections
         orchestration_stacks_parameters
       )
     )
+
+    add_inventory_collection(
```

Comment:
@mansam nice, best to add it for orchestration stack too, without it the Stack are doing n+1 requests.

Btw. the history of this optimization is, that before we were keeping active record object around, so .parent = obj worked fine. But AR objects take a lot of space in memory, so we ditched that and now we keep only :id, that means each .parent = obj requires fetching it from the DB again, since Ancestry requires to use AR objects and can't work with just :id, so that is why we have the custom saving methods, that will effectively load it in batches as set it.

So the root of the problem is that Ancestry is not Rails friendly, since it always require 2 AR objects to fill the parent. Hopefully this might be fixed in the future.